tasks/uk/gov.py

```diff
@@ -0,0 +1,447 @@
+from tasks.base_tasks import (ColumnsTask, TableTask, TagsTask, DownloadUnzipTask, GeoFile2TempTableTask,
+                              SimplifiedTempTableTask, RepoFile)
+from tasks.tags import SectionTags, SubsectionTags, LicenseTags, BoundaryTags
+from tasks.meta import GEOM_REF, OBSColumn, OBSTable, OBSTag, current_session
+from tasks.util import copyfile
+import os
+from luigi import WrapperTask
+from collections import OrderedDict
+from lib.timespan import get_timespan
+
+
+class LowerLayerSuperOutputAreasSourceTags(TagsTask):
+
+    def version(self):
+        return 1
+
+    def tags(self):
+        return[
+            OBSTag(id='uk-gov-source',
+                   name='Office for National Statistics (UK)',
+                   type='source',
+                   description='Lower Layer Super Output Area (LSOA) and Data Zones boundaries')
+                   ]
+
+
+class DownloadLowerLayerSuperOutputAreas(DownloadUnzipTask):
+    # https://data.gov.uk/dataset/fa883558-22fb-4a1a-8529-cffdee47d500/lower-layer-super-output-area-lsoa-boundaries
+    # http://geoportal.statistics.gov.uk/datasets/da831f80764346889837c72508f046fa_0
+    URL = 'https://opendata.arcgis.com/datasets/da831f80764346889837c72508f046fa_0.zip?outSR=%7B%22wkid%22%3A27700%2C%22latestWkid%22%3A27700%7D'
+
+    def version(self):
+        return 1
+
+    def requires(self):
```

Comment:
It's not that easy because `DownloadUnzipTask`s require `RepoFile` (and I have just realised that we don't need the `download` method anymore). It's worth thinking it slowly, I'm adding a new issue:  https://github.com/CartoDB/bigmetadata/issues/560